components/sup/src/manager/service_updater.rs

```diff
@@ -400,4 +399,21 @@ impl Worker {
         let pkg = PackageInstall::load(archive.ident().as_ref().unwrap(), Some(&*FS_ROOT_PATH))?;
         Ok(pkg)
     }
+
+    fn update_frequency(&self) -> i64 {
+        match env::var(FREQUENCY_ENVVAR) {
+            Ok(val) => {
+                match val.parse::<i64>() {
+                    Ok(num) => num,
+                    Err(_) => {
+                        outputln!("Unable to parse {} from {} as a valid integer",
```

Comment:
Might want to explain that we were trying to parse the update frequency and from which env var here so people know what was not a valid integer ;)